aten/src/ATen/record_function.h

```diff
@@ -447,6 +447,14 @@ struct TORCH_API RecordFunction {
 #endif
   }
 
+  std::string getKernelFile() const {
+    return kernel_file_;
+  }
+
+  void setKernelFile(const std::string& kernel_file) {
+    kernel_file_ = kernel_file;
+  }
```

Comment:
NIT: Can avoid a few copies
```suggestion
  const std::string& getKernelFile() const {
    return kernel_file_;
  }

  void setKernelFile(std::string kernel_file) {
    kernel_file_ = std::move(kernel_file);
  }
```